lib/product.rb

```diff
@@ -0,0 +1,27 @@
+class Product
+  attr_reader :category,
+              :name,
+              :unit_price,
+              :quantity,
+              :hoarded
```

Comment:
Since we have a method `is_hoarded?` to get the value of this instance variable, we do not also need an attr_reader.